pkgs/applications/altcoins/ledger-live-desktop/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+ stdenv,
+ appimage-run,
+ fetchurl,
+ runtimeShell,
+}:
+
+stdenv.mkDerivation rec {
+  github-user = "LedgerHQ";
+  pname = "ledger-live-desktop";
+  version = "1.9.1";
+
+  src = fetchurl {
+    url = "https://github.com/${github-user}/${pname}/releases/download/v${version}/${pname}-${version}-linux-x86_64.AppImage";
```

Comment:
```suggestion
    url = "https://github.com/LedgerHQ/${pname}/releases/download/v${version}/${pname}-${version}-linux-x86_64.AppImage";
```

Reply:
@bhipple why would you do this?

Reply:
I don't see how splitting the github-user to a separate variable makes anything neater

Reply:
@alexarice i like it, easier copypasta for the next app image i submit ;)

Reply:
Possibly, but it's not a convention used elsewhere in NixPkgs, and there's some value in consistency given how huge the distribution is.  

Another consideration is that attributes at the top-level of the derivation, such as `buildInputs` or `pname` -- or your `github-user` -- make it into the hash and become part of the propagated description of the package in the derivation.  E.g., it'll show up in `nix show-derivation`, and if the github repo moves/changes names without changing src it'll change the hash and cause a rebuild.  

Not a very big deal and mostly a bike shed, but personally I'd still recommend removing it.